Conch/cpanfile

```diff
@@ -46,6 +46,7 @@ requires 'String::CamelCase', '>= 0.04';
 
 # database
 requires 'DBD::Pg';
+requires 'DateTime::Format::Pg';
```

Comment:
I thought we decided to not use this module any more. I'm not seeing where it's used in the PR either.

Reply:
I got an exception for this module missing during evaluation of the old validation logic. It's still used as part of the DBIC classes in Conch::Legacy::Schema::* through DBIx::Class::InflateColumn::DateTime.

We may not have seen this exception in production because DateTime::Format::Pg is still in the `local` cache from previous deployments.  I had recently blown away my `local` dir when it occurred

I'll reproduce the issue and add comments.

Reply:
I removed this dependency, deleted my `local` dir, `carton install`, and processed a report. It threw an exception missing the dependency.

```
[Thu Apr  5 09:22:59 2018] [error] Can't locate DateTime/Format/Pg.pm in @INC (you may need to install the DateTime::Format::Pg module) (@INC contains: /Users/lsep/dev/joyent/conch/Conch/bin/../lib /Users/lsep/dev/joyent/conch/Conch/local/lib/perl5/darwin-thread-multi-2level /Users/lsep/dev/joyent/conch/Conch/local/lib/perl5 /usr/local/Cellar/perl/5.26.1/lib/per
l5/site_perl/5.26.1/darwin-thread-multi-2level /usr/local/Cellar/perl/5.26.1/lib/perl5/site_perl/5.26.1 /usr/local/Cellar/perl/5.26.1/lib/perl5/5.26.1/darwin-thread-multi-2level /usr/local/Cellar/perl/5.26.1/lib/perl5/5.26.1 /usr/local/lib/perl5/site_perl/5.26.1) at /Users/lsep/dev/joyent/conch/Conch/local/lib/perl5/Class/C3/Component
ised.pm line 146.
141:     return if ( *{"${f_class}::$_"}{CODE} );
142:   }
143:
144:   # require always returns true on success
145:   # ill-behaved modules might very well obliterate $_
146:   eval { local $_; require($file) } or do {
147:
148:     $@ = "Invalid class name '$f_class'" if $f_class =~ $invalid_class;
149:
150:     if ($class->can('throw_exception')) {
151:       $class->throw_exception($@);

[Thu Apr  5 09:22:59 2018] [error] Stack Trace (first 10 frames):
        Mojolicious::__ANON__ at /Users/lsep/dev/joyent/conch/Conch/local/lib/perl5/Class/C3/Componentised.pm:146
        (eval) at /Users/lsep/dev/joyent/conch/Conch/local/lib/perl5/Class/C3/Componentised.pm:146
        Class::C3::Componentised::ensure_class_loaded at /Users/lsep/dev/joyent/conch/Conch/local/lib/perl5/DBIx/Class/AccessorGroup.pm:19
        DBIx::Class::AccessorGroup::get_component_class at (eval 1156):2
        DBIx::Class::Storage::DBI::datetime_parser_type at /Users/lsep/dev/joyent/conch/Conch/local/lib/perl5/DBIx/Class/Storage/DBI.pm:3141
        DBIx::Class::Storage::DBI::build_datetime_parser at /Users/lsep/dev/joyent/conch/Conch/local/lib/perl5/DBIx/Class/Storage/DBI.pm:3124
        DBIx::Class::Storage::DBI::datetime_parser at /Users/lsep/dev/joyent/conch/Conch/local/lib/perl5/DBIx/Class/InflateColumn/DateTime.pm:221
        DBIx::Class::InflateColumn::DateTime::_datetime_parser at /Users/lsep/dev/joyent/conch/Conch/local/lib/perl5/DBIx/Class/InflateColumn/DateTime.pm:196
        DBIx::Class::InflateColumn::DateTime::_flate_or_fallback at /Users/lsep/dev/joyent/conch/Conch/local/lib/perl5/DBIx/Class/InflateColumn/DateTime.pm:212
        DBIx::Class::InflateColumn::DateTime::_inflate_to_datetime at /Users/lsep/dev/joyent/conch/Conch/local/lib/perl5/DBIx/Class/InflateColumn/DateTime.pm:175
        DBIx::Class::InflateColumn::DateTime::__ANON__ at /Users/lsep/dev/joyent/conch/Conch/local/lib/perl5/DBIx/Class/InflateColumn.pm:122
```

Oddly, DateTime formats for SQLite and MySQL are pre-requistes for  DBIx::Class::TimeStamp and installed, but not for Postgres.

```
...
Successfully installed DateTime-Format-SQLite-0.11
Successfully installed DateTime-Format-MySQL-0.06
...
```